src/routes/plans/[id]/+page.svelte

```diff
@@ -259,7 +259,7 @@
 
   function onCloseSnapshotPreview() {
     $planSnapshotId = null;
-    $simulationDatasetId = -1;
+    $simulationDatasetId = data.initialPlan.simulations[0]?.simulation_datasets[0]?.id ?? -1;
```

Comment:
One potential issue here is that any new sims that happened post page load won't be in this list, could instead use the planSnapshotsWithSimulations store?

Reply:
Good point. I can't use `planSnapshotsWithSimulations` because they may not be viewing a snapshot. I did update it to use `$simulationDatasetLatest`. I think that's what we want.